pyproject.toml

```diff
@@ -11,7 +11,7 @@ visivo = "visivo.command_line:safe_visivo"
 
 [tool.poetry.dependencies]
 python = " >=3.10,<3.13"
-pydantic = ">=2.5.0"
+pydantic = ">=2.5.0, <=2.9.0"
```

Comment:
?

Reply:
It seems a little risky to let our pydantic dependency upgrade unbounded without validating that new versions of pydantic don't break Visivo